### DISCUSSION TITLE
Benchmark compared to other engines

### DISCUSSION THREAD

**Discussion by gkorland** [boa-dev/boa#2477] [General]
I saw the https://boa-dev.github.io/boa/dev/bench/ link to the benchmark.
Did anyone try to compare against other embeddable engine?
e.g. 
https://bellard.org/quickjs/bench.html

Meant using v8-bench https://github.com/mozilla/arewefastyet/tree/master/benchmarks/v8-v7

**Comment by Razican** [comment]
We have #1924 open regarding this

**Comment by jedel1043** [comment]
Resolving since #1924 is closed.
